rasr/feature_scorer.py

```diff
@@ -126,25 +127,51 @@ def __init__(self, prior_mixtures, scale=1.0, priori_scale=0.0, prior_file=None)
         self.config.normalize_mixture_weights = False
 
 
-class TFLabelContextFeatureScorer(FeatureScorer):
+class OnnxFeatureScorer(FeatureScorer):
     def __init__(
         self,
-        fs_tf_config,
-        contextPriorFile,
-        diphonePriorFile,
-        prior_mixtures,
-        prior_scale,
+        mixtures,
+        model,
+        io_map,
+        *args,
+        label_log_posterior_scale=1.0,
+        label_prior_scale=0.7,
+        label_log_prior_file=None,
+        apply_log_on_output=False,
+        negate_output=True,
+        intra_op_threads=1,
+        inter_op_threads=1,
+        **kwargs,
     ):
-        super().__init__()
+        """
+        :param str mixtures: path to a *.mix file e.g. output of either EstimateMixturesJob or CreateDummyMixturesJob
+        :param str model: path of a model e.g. output of ExportPyTorchModelToOnnxJob
+        :param dict io_map: mapping between internal rasr identifiers and the model related input/output. Default key values
+                            are "features" and "output", and optionally "features-size"
+        :param float label_log_posterior_scale: scales for the log probability of a label e.g. 1.0 is recommended
+        :param float label_prior_scale: scale for the prior log probability of a label reasonable e.g. values in [0.1, 0.7] interval
+        :param str label_log_prior_file: xml file containing log prior probabilities e.g. estimated from the model via povey method
+        :param bool apply_log_on_output: whether to apply the log-function on the output, usefull if the model outputs softmax instead of log-softmax
+        :param bool negate_output: wheter negate output (because the model outputs log softmax and not negative log softmax
+        """
```

Comment:
Please also add the entries for the op_threads, you can e.g. just point to:

 https://onnxruntime.ai/docs/performance/tune-performance/threading.html

Reply:
Oh, I meant you can just but the link, but this is also fine.